pkg/controller/virtualmachineimport/virtualmachineimport_controller.go

```diff
@@ -121,21 +129,38 @@ func (r *ReconcileVirtualMachineImport) Reconcile(request reconcile.Request) (re
 	if err != nil {
 		return reconcile.Result{}, err
 	}
-	ovirtConnection, err := createOvirtConnection(ovirtSecret)
+
+	ovirt, err := ovirtclient.NewRitchOvirtClient(
+		&ovirtclient.ConnectionSettings{
+			URL:      string(ovirtSecret.Data["apiUrl"]),
```

Comment:
The ovirt secret should look obtain data from `ovirt` section according to the agreed format of the secret. See example on README: https://github.com/kubevirt/vm-import-operator/blob/master/README.md

I updated the example here: https://github.com/kubevirt/vm-import-operator/pull/16

Reply:
Thanks, done.